shared/tracker/index.js

```diff
@@ -47,9 +47,28 @@ export function trackerPropsToRenderProps ({currentlyFollowing, inviteLink, isPr
     proofs, reason, serviceName, trackerState, userInfo, username, waiting, loading, actionBarReady}:
 TrackerProps): RenderPropsUnshaped {
   return {
-    currentlyFollowing, inviteLink, isPrivate, lastAction,
-    loggedIn, name, nonUser, onClose, onFollow, onIgnore, onRefollow, onUnfollow, parentProps,
-    proofs, reason, serviceName, trackerState, userInfo, username, waiting, loading, actionBarReady,
+    actionBarReady,
+    currentlyFollowing,
+    inviteLink,
+    isPrivate,
+    lastAction,
+    loading,
+    loggedIn,
+    name,
+    nonUser,
+    onClose,
+    onFollow,
+    onIgnore,
+    onRefollow,
+    onUnfollow,
+    parentProps,
+    proofs,
+    reason,
+    serviceName,
+    trackerState,
+    userInfo,
+    username,
```

Comment:
I think it actually helps us point out code smells. Like really this
function should be deprecated

On Friday, August 19, 2016, Chris Ball <notifications@github.com> wrote:

> In shared/tracker/index.js
> <https://github.com/keybase/client/pull/3972#discussion_r75564795>:
>
> > +    loading,
> > +    loggedIn,
> > +    name,
> > +    nonUser,
> > +    onClose,
> > +    onFollow,
> > +    onIgnore,
> > +    onRefollow,
> > +    onUnfollow,
> > +    parentProps,
> > +    proofs,
> > +    reason,
> > +    serviceName,
> > +    trackerState,
> > +    userInfo,
> > +    username,
>
> Ugh I hate this file's change. But it looks like it's the only one like
> this, so I'll hold my nose and 👍
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3972/files/6bc8ea0d2ad64e9bc5b601e8d21d72c0170f0096#r75564795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU9i2Jyr6q1HkffN3TDYlkY7So1pyks5qhjuYgaJpZM4Jo4ag>
> .
>